common/static/common/js/discussion/views/discussion_inline_view.js

```diff
@@ -118,7 +118,7 @@
             this.newPostView = new NewPostView({
                 el: this.newPostForm,
                 collection: this.discussion,
-                course_settings: this.course_settings,
+                course_settings: this.courseSettings,
```

Comment:
It will be good if we can follow camel case convention here also. Only if we don't have to refactor a lot of other code.